plugins/kobolight.koplugin/main.lua

```diff
@@ -148,4 +159,39 @@ function KoboLight:onSwipe(_, ges)
     return true
 end
 
+function KoboLight:addToMainMenu(tab_item_table)
+    table.insert(tab_item_table.plugins, {
+        text = _("Frontlight gesture controller"),
+        callback = function()
+            local image = ImageWidget:new{
+                file = PluginLoader.plugin_path .. "/kobolight.koplugin/demo.png",
+                height = Screen:getHeight(),
+                width = Screen:getWidth(),
+                scale_factor = 0,
+            }
+            UIManager:show(image)
+            UIManager:show(ConfirmBox:new{
+                text = T(_("Frontlight gesture controller can:\n- Turn on or off frontlight by tapping left-bottom of the screen.\n- Change frontlight intensity by swiping up or down on the left of the screen.\n\nDo you want to %1 it?"),
+                         self:disabled() and _("enable") or _("disable")),
+                ok_text = self:disabled() and _("Enable") or _("Disable"),
```

Comment:
I feel like we can either remove `Do you want to %1 it` or rename this ok_text to "Yes". It seems like a duplication if we keep both.